wordpress_importer/management/commands/import_from_wordpress.py

```diff
@@ -651,7 +651,7 @@ def process_html_for_series_links(self, html, series):
                 potential_article_slug = path_parts[-1]
                 try:
                     article = ArticlePage.objects.get(slug=potential_article_slug)
-                    InDepthArticleLink.objects.create(article=article, in_depth=series)
+                    InDepthArticleLink.objects.get_or_create(article=article, in_depth=series)
```

Comment:
These are unrelated changes that have been pulled in, but aren't associated with another issue.